src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,169 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> addedElement = new Node<>(null, value, null);
```

Comment:
```suggestion
        Node<T> newElement = new Node<>(null, value, null);
```

We haven't added it yet